packages/functional-tests/src/app.ts

```diff
@@ -87,4 +85,56 @@ export default class App {
             delete this.activeTests[testName];
         }
     }
+
+
+    private async launchTestBrowser() {
+        while (true) {
+            const tester = MRESDK.Actor.CreateEmpty(this.context, {});
+            var selectedTestName = undefined;

```

Comment:
Did you mean to use `var` here? This should have been flagged by tslint. Also, no need to initialize to `undefined`. It's the default.